src/gd_kotlin.cpp

```diff
@@ -243,13 +178,13 @@ void GDKotlin::init() {
                     jvm_jmx_port = "9010";
                 }
             }
-        } else if (cmd_arg.find("--jvm-to-engine-shared-buffer-size") >= 0) {
+        } else if (cmd_arg.find("--jvm_to_engine_max_string_size") >= 0) {
```

Comment:
Other command lines argument are with `-`. For coherence, this should be `--jvm-to-engine-max-string-size`